bank_atm.rb

```diff
@@ -0,0 +1,116 @@
+$LOAD_PATH << '.'
+require 'validations'
+require_relative 'models/initializer'
+
+#Begin of demo data
```

Comment:
Usually for comments, we should add a space after `#`

Reply:
Also, instead of adding comments for the demo data setup, you could extract this into a `setup_demo_data` method, thus you would write code that comments itself :D